common/hexutil/hexutil.go

```diff
@@ -238,3 +238,17 @@ func mapError(err error) error {
 	}
 	return err
 }
+
+// TrimLeftZeroes returns a subslice of s without leading zeroes
+func TrimLeftZeroes(s []byte) []byte {
+	idx := 0
+	for ; idx < len(s); idx++ {
```

Comment:
for ; idx < len(s) && idx[s] == 0; idx++ {

Reply:
Myeah, suprt readable :)

Reply:
.... ignored...